lib/config/validation.ts

```diff
@@ -787,6 +775,139 @@ function validateRegexManagerFields(
   }
 }
 
+/**
+ * Basic validation for global config options
+ */
+function validateGlobalConfig(
```

Comment:
I don't know about you maintainers, but the code for the `validateGlobalConfig` function is hard to read for me, because of all the `if` and `else if` statements.

Maybe you can extract some blocks of code into separate validation functions?

Reply:
yes, I think this should be modelled like the migration service, so we've small validation functions which can be easily tested.

Reply:
I think this should be done seperately under its own issue, will create one